src/plugins/rel/rel.js

```diff
@@ -196,6 +213,23 @@
                 } else {
                     step.el.setAttribute( "data-z", step.z );
                 }
+                if ( step.rotateX === null ) {
+                    step.el.removeAttribute( "data-rotate-x" );
+                } else {
+                    step.el.setAttribute( "data-rotate-x", step.rotateX );
+                }
+                if ( step.rotateY === null ) {
+                    step.el.removeAttribute( "data-rotate-y" );
+                } else {
+                    step.el.setAttribute( "data-rotate-y", step.rotateY );
+                }
+                if ( step.rotateZ === null ) {
+                    step.el.removeAttribute( "data-rotate-z" );
+                    step.el.removeAttribute( "data-rotate" );
+                } else {
+                    step.el.setAttribute( "data-rotate-z", step.rotateZ );
+                    step.el.setAttribute( "data-rotate", step.rotateZ );
```

Comment:
The point here is to return the initial state. You can't set both data-rotate and data-rotate-z, rather you must set them exactly as they were when the html was loaded. (Or rather, at the time that impress().init() was called.)